pkg/commands/oscommands/cmd_obj_runner.go

```diff
@@ -90,6 +116,20 @@ func (self *cmdObjRunner) RunWithOutputAux(cmdObj ICmdObj) (string, error) {
 	return output, err
 }
 
+func (self *cmdObjRunner) RunWithStdoutAux(cmdObj ICmdObj) (string, error) {
+	self.log.WithField("command", cmdObj.ToString()).Debug("RunCommand")
+
+	if cmdObj.ShouldLog() {
+		self.logCmdObj(cmdObj)
+	}
+
+	output, err := sanitisedCommandOutput(cmdObj.GetCmd().Output())
```

Comment:
@Ryooooooga if there actually is an error that we want to surface to the user, will this properly capture it?

Reply:
ah. Changed to capture error messages and return stderr as well.